packages/firestore/src/platform/browser_lite/fetch_connection.ts

```diff
@@ -69,9 +69,23 @@ export class FetchConnection extends RestConnection {
     }
 
     if (!response.ok) {
+      let errorMessage;
+      try {
+        // Try to get any error message text from server response.
+        //response.json returns an error object inside an array.
```

Comment:
Remove these comments since they are just explaining what the code is clearly doing. Also on line 87.